notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1375 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    train_job = aiplatform.AutoMLTabularTrainingJob(
</pre><p>Please add few points explaining the params as shown in the regression notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>done. Also, removed the default parameters and added reference to the docs.</p>

Reply:
<p>Thank you!</p>